packages/frontend/src/components/wallet-migration/WalletMigration.jsx

```diff
@@ -91,22 +92,30 @@ const WalletMigration = ({ open, onClose }) => {
         handleSetActiveView(WALLET_MIGRATION_VIEWS.VERIFYING);
     };
 
-    const navigateToCleanKeys = () => {
-        handleSetActiveView(WALLET_MIGRATION_VIEWS.CLEAN_KEYS);
-    };
-
     const navigateToLogOut = () => {
         setMigrationStep(WALLET_MIGRATION_VIEWS.LOG_OUT);
         handleSetActiveView(WALLET_MIGRATION_VIEWS.LOG_OUT);
     };
 
+    const navigateToMigrateAccounts = () => {
+        handleSetActiveView(WALLET_MIGRATION_VIEWS.MIGRATE_ACCOUNTS);
+    };
+
     const onLogout = () => {
-        // TODO: Add logic to remove FAK(s) here so everything gets cleared together
-        return Promise.all(availableAccounts.map((accountId) => wallet.removeWalletAccount(accountId)))
+        setIsLoggingOut(true);
+        return Promise.all(availableAccounts.map(async (accountId) => {
+            const account = await wallet.getAccount(accountId);
+            const keyPair = await wallet.keyStore.getKey(wallet.connection.networkId, accountId);
+            await account.signAndSendTransaction({
+                actions: [nearApi.transactions.deleteKey(keyPair.publicKey)],
+                receiverId: accountId,
+            });
```

Comment:
We have some shorthand around this already that I think would be better to use here:
```js
            const publicKey = await wallet.getPublicKey(accountId);
            await account.deleteKey(publicKey);
```